pom.xml

```diff
@@ -144,6 +144,7 @@
     <jackson.version>2.8.11</jackson.version>
     <jsch.version>0.1.54</jsch.version>
     <jetty.version>6.1.22</jetty.version>
+    <jetty.test.version>9.4.50.v20221201</jetty.test.version>
```

Comment:
Seems like we cannot simply upgrade `jetty.version` in parent pom, the `org.mortbay.jetty` is not available. Since we only need 9.4.x for scm test purpose, I added one property called `jetty.test.version` here